Author: Joost Yervante Damad <andete@debian.org>
Description: Fix off-by-one crash error in panning reverb
Bug-Debian: https://bugs.debian.org/440543

--- a/timidity/reverb.c
+++ b/timidity/reverb.c
@@ -1640,8 +1640,8 @@
 		buf[i] += r;
 		buf[++i] += l;
 
-		if (++index0 == buf_size) {index0 = 0;}
-		if (++buf_index == buf_size) {buf_index = 0;}
+		if (index0++ == buf_size) {index0 = 0;}
+		if (buf_index++ == buf_size) {buf_index = 0;}
 	}
 	memset(reverb_effect_buffer, 0, sizeof(int32) * count);
 	info->index[0] = index0;
